Logger/BaseLogger.cs

```diff
@@ -2,6 +2,8 @@
 {
     public abstract class BaseLogger
     {
+        public string name { get; set; }
```

Comment:
Properties should be PascalCase
```suggestion
        public string Name { get; set; }
```